docs/concepts/models.md

```diff
@@ -250,107 +268,61 @@ whole model (nested models and all), so all types at all levels need to be ready
 
 ## Arbitrary class instances
 
-(Formerly known as "ORM Mode"/`from_orm`.)
+(Formerly known as "ORM Mode"/`from_orm`).
 
 Pydantic models can also be created from arbitrary class instances by reading the instance attributes corresponding
 to the model field names. One common application of this functionality is integration with object-relational mappings
 (ORMs).
 
-To do this, set the config attribute `model_config['from_attributes'] = True`. See
-[Model Config][pydantic.config.ConfigDict.from_attributes] and [ConfigDict][pydantic.config.ConfigDict] for more information.
+To do this, set the [`from_attributes`][pydantic.config.ConfigDict.from_attributes] config value to `True`
+(see the documentation on [Configuration](./config.md) for more details).
 
 The example here uses [SQLAlchemy](https://www.sqlalchemy.org/), but the same approach should work for any ORM.
 
 ```py
 from typing import List
 
-from sqlalchemy import Column, Integer, String
-from sqlalchemy.dialects.postgresql import ARRAY
-from sqlalchemy.orm import declarative_base
+from sqlalchemy import ARRAY, String
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
 from typing_extensions import Annotated
 
 from pydantic import BaseModel, ConfigDict, StringConstraints
 
-Base = declarative_base()
+
+class Base(DeclarativeBase):
+    pass
 
 
 class CompanyOrm(Base):
     __tablename__ = 'companies'
 
-    id = Column(Integer, primary_key=True, nullable=False)
-    public_key = Column(String(20), index=True, nullable=False, unique=True)
-    name = Column(String(63), unique=True)
-    domains = Column(ARRAY(String(255)))
+    id: Mapped[int] = mapped_column(primary_key=True, nullable=False)
+    public_key: Mapped[str] = mapped_column(
+        String(20), index=True, nullable=False, unique=True
+    )
+    domains: Mapped[List[str]] = mapped_column(ARRAY(String(255)))
 
 
 class CompanyModel(BaseModel):
     model_config = ConfigDict(from_attributes=True)
 
     id: int
     public_key: Annotated[str, StringConstraints(max_length=20)]
-    name: Annotated[str, StringConstraints(max_length=63)]
     domains: List[Annotated[str, StringConstraints(max_length=255)]]
 
 
 co_orm = CompanyOrm(
     id=123,
     public_key='foobar',
-    name='Testing',
     domains=['example.com', 'foobar.com'],
 )
 print(co_orm)
 #> <__main__.CompanyOrm object at 0x0123456789ab>
 co_model = CompanyModel.model_validate(co_orm)
 print(co_model)
-"""
-id=123 public_key='foobar' name='Testing' domains=['example.com', 'foobar.com']
-"""
-```
-
-### Reserved names
```

Comment:
I decided to delete this section, as it seems overly specific and similar information can be found in the aliases docs.

Reply:
Agreed that this can be removed - could you add this (bc it's in the context of a sqlalchemy pattern) to the examples section instead?

Reply:
Let's create an issue related to doc examples and SQLAlchemy. I'll link this comment for reference so that we can tackle this properly. wdyt?

Reply:
Sounds great - could you create this issue + link this one!

Reply:
Created https://github.com/pydantic/pydantic/issues/10251